domain/application/service/charm.go

```diff
@@ -657,9 +657,6 @@ func (s *Service) SetCharmAvailable(ctx context.Context, locator charm.CharmLoca
 // state.
 // If there are any non-blocking issues with the charm metadata, actions,
 // config or manifest, a set of warnings will be returned.
-//
-// Deprecated: Use a feature specific method to set the charm. This method
-// should become private in the future.
```

Comment:
No longer deprecated?

Reply:
No, I think there is a valid use case. Maybe we should rename it?